pkg/quota/apiserver/registry/clusterresourcequota/etcd/etcd.go

```diff
@@ -70,10 +71,10 @@ func (r *StatusREST) New() runtime.Object {
 
 // Get retrieves the object from the storage. It is required to support Patch.
 func (r *StatusREST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	return r.store.Get(ctx, name, options)
+	return nil, fmt.Errorf("clusterresourcequota is no longer openshift apiserver object")
```

Comment:
You need to replace `NewREST` with something that fulfills the interface and returns errors for every method.

Reply:
updated